app-portage/emlop/emlop-0.4.1.ebuild

```
@@ -0,0 +1,108 @@
+# Copyright 2017-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Auto-Generated by cargo-ebuild 0.3.1
+
+EAPI=7
+
+CRATES="
+aho-corasick-0.7.15
```

IIRC newer vesions of cargo-ebuild intent the CRATES with a tab.
```suggestion
	aho-corasick-0.7.15
```
(and so on)

---

Indeed. I'll regenerate with latest cargo-ebuild.